plugins/modules/junos_facts.py

```diff
@@ -68,6 +68,10 @@
     required: false
     type: list
     elements: str
+  available_network_resources:
+    description: This key allows to return the facts only with available resources list.
```

Comment:
--->

When 'True' a list of network resource for which resource modules are available will be provided.